core/src/test/java/org/neo4j/ogm/persistence/examples/cineasts/annotated/CineastsIntegrationTest.java

```diff
@@ -253,21 +250,20 @@ public void shouldModifyStringArraysCorrectly() throws MalformedURLException {
      */
     @Test
     public void shouldBeAbleToSetNodePropertiesToNull() throws MalformedURLException {
-        Movie movie = new Movie();
-        movie.setTitle("Zootopia");
+        Movie movie = new Movie("Zootopia", 2016);
         movie.setImdbUrl(new URL("http://www.imdb.com/title/tt2948356/"));
         session.save(movie);
 
         movie.setTitle(null);
         movie.setImdbUrl(null);
         session.save(movie);
 
-        movie = session.load(Movie.class, movie.getId());
+        movie = session.load(Movie.class, movie.getUuid());
```

Comment:
A minor concern: by refactoring the Movie class to have a primary index, have we potentially obscured other issues relating to specific tests of id field elsewhere? (Presumably all tests that used `movie.getId()` now use `movie.getUuid()` instead)